<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Directorios</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QListWidget" name="Explorer1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>256</width>
     <height>361</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="Explorer2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>260</width>
     <height>361</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="NDir1">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>440</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Nueva Carpeta</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>newFolder.ico</normaloff>newFolder.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="NDir2">
   <property name="geometry">
    <rect>
     <x>651</x>
     <y>440</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Nueva Carpeta</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>newFolder.ico</normaloff>newFolder.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="NFile2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>440</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Nuevo Archivo</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>newFile.ico</normaloff>newFile.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>440</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Eliminar</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>trash.ico</normaloff>trash.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="LeftToRigth">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-----&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RigthToLeft">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-----</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NFile1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Nuevo Archivo</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>newFile.ico</normaloff>newFile.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Eliminar</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>trash.ico</normaloff>trash.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Directorio 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Directorio 2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
